jobs/aws/eks-anywhere-build-tooling/harbor-presubmits.yaml

```diff
@@ -56,9 +56,11 @@ presubmits:
           requests:
```

Comment:
Side Note, if you put the same `requests` as `limits`, you can just omit the `requests` portion. 

If you put 
```
    resources:
      limits:
        memory: "1Gi"
```

it would converts into
```
resources:
  limits:
    memory: 1Gi
  requests:
    memory: 1Gi
```